src/components/VotableTags/VotableTags.tsx

```diff
@@ -30,9 +30,9 @@ export function VotableTags({
 }: GalleryTagProps) {
   const queryUtils = trpc.useContext();
   const setVote = useVotableTagStore((state) => state.setVote);
-  const { data: tags = initialTags, isLoading } = trpc.tag.getVotableTags.useQuery(
+  const { data: tags = [], isLoading } = trpc.tag.getVotableTags.useQuery(
     { id, type },
-    { enabled: !initialTags }
+    { enabled: !initialTags, initialData: initialTags }
```

Comment:
Not sure if it's intentional (looking at the change seems like it is) but using `initialData` here would make it so that whenever `getVotableTags` query resolves for the first time, the tags won't be updated with the server data until the query gets invalidated or its `staleTime` has expired. Maybe we can use `placeholderData` instead? I could be wrong, and your approach is what we are looking for here

Reply:
My understanding is that by setting `initialData`, I am giving react-query data to cache as if it had already fetched the data. I want to bypass the need to fetch data by giving it the data it needs directly through props.

Reply:
Yeah that's what I thought too, good to know this is the right approach